pkg/client/daemon/service.go

```diff
@@ -48,7 +48,6 @@ to troubleshoot problems.
 // service represents the state of the Telepresence Daemon
 type service struct {
 	rpc.UnsafeDaemonServer
-	dns           string
```

Comment:
This change doesn't seem directly related to the others, but was this just an extra value that we weren't using? I guess I'd expect more changes if we were removing a value from the struct, so just want to make sure I understand.

Reply:
Yes, it's a removal of a field that became obsolete some time ago.